src/storage.ts

```diff
@@ -1,11 +1,17 @@
-const BASE_DIR = 'kv'
-
-export const set = (key: string, value: any) => {
-  // writeFileSync(BASE_DIR + '/' + key, JSON.stringify(value))
-  return true
+export const set = async (
+  KV: KVNamespace,
+  context: ExecutionContext,
+  key: string,
+  value: any
+) => {
+  if (value) {
+    await KV.put(key, value)
```

Comment:
let's do something like this:
```
const promise = KV.put(key, value)
waitUntil(promise)
await promise
```

to make sure that the promise will be executed even if the MC didn't `await` their `set` call